tests/integration/README.md

```
@@ -1,4 +1,55 @@
+
+Tests are declared as a set of actions performed by a set of external actors.  For example a test might specify that identity 1 creates a collection, then adds a document.
+
+Each action is a reusable data struct, that matches as closely as practical to how an actor might interact with the database.
+
+Actions are declared ahead of time, and only executed after all actions have been declared and modified by `complexity multipliers`.
+
+Many of the actions have been migrated to `/tests/action`, this allows the actions to also be used to write benchmark tests. 
+
+### Complexity multipliers
+
+The Defra integration test suite evolved organically into something that conceptually matched (testo)[https://github.com/sourcenetwork/testo/] complexity multipliers, but the core suite has not yet been migrated to the testo system.  The CLI integration tests, Lens, and corekv tests do make direct use of testo.
```

markdownlint-cli2 (0.22.0)

[warning] Reversed link syntax
(testo)